doc/holochain_101/src/zome/api_functions.md

```diff
@@ -137,39 +137,47 @@ Entry lookup is done in the following order:
 Caller can request additional metadata on the entry such as type or sources
 (hashes of the agents that committed the entry).
 
-[View it in the Rust HDK](https://developer.holochain.org/api/latest/hdk/api/fn.get_entry.html)
+- [View get_entry in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry.html)
+- [View get_entry_initial in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry_initial.html)
+- [View get_entry_initial in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry_initial.html)
+- [View get_entry_result in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry_result.html)
+
 
 ### Get Links
 
 Canonical name: `get_links`
 
 Consumes two values, the first of which is the address of an entry, base, and the second of which is a string, tag, used to describe the relationship between the base and other entries you wish to lookup. Returns a list of addresses of other entries which matched as being linked by the given tag. Links are created in the first place using the Zome API function [link_entries](#link-entries). Once you have the addresses, there is a good likelihood that you will wish to call [get_entry](#get-entry) for each of them.
 
-[View it in the Rust HDK](https://developer.holochain.org/api/latest/hdk/api/fn.get_links.html)
+- [View get_links in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_links.html)
+- [View get_links_and_load in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_links_and_load.html)
+- [View get_links_result in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_links_result.html)
+- [View get_links_with_options in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_links_with_options.html)
+
 
 ### Link Entries
 
 Canonical name: `link_entries`
 
 Consumes three values, two of which are the addresses of entries, and one of which is a string that defines a relationship between them, called a `tag`. Later, lists of entries can be looked up by using `get_links`. Entries can only be looked up in the direction from the `base`, which is the first argument, to the `target`, which is the second.
 
-[View it in the Rust HDK](https://developer.holochain.org/api/latest/hdk/api/fn.link_entries.html)
+[View it in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.link_entries.html)
 
 ### Query
 
 Canonical name: `query`
 
 Returns a list of addresses of entries from your local source chain, that match a given entry type name, or a vector of names. You can optionally limit the number of results, and you can use "glob" patterns such as "prefix/*" to specify the entry type names desired.
```

Comment:
looks like there's an unescaped asterisk here which is making the rest of the line italic. Not related to the links updates, sorry, but thought I'd mention it while it was in front of me :)